lib/AbstractHandle/Utils/MongoUtil.py

```diff
@@ -60,34 +62,26 @@ def __init__(self, config):
         self.mongo_user = config['mongo-user']
         self.mongo_pass = config['mongo-password']
         self.mongo_authmechanism = config['mongo-authmechanism']
+        self.mongo_retrywrites = config['mongo-retrywrites']
```

Comment:
```suggestion
        self.mongo_retrywrites = config.get('mongo-retrywrites) == "true"
```

Reply:
I think this processing step should be done in get_config()

Reply:
get_config is in test code. Ir won't be run under normal operations

Reply:
ohhhh, how can I miss this

Reply:
👍

Reply:
:+1: